### DISCUSSION TITLE
Using udisksctl instead of pmount in nmount plugin.

### DISCUSSION THREAD

**Discussion by TheUtopian** [jarun/nnn#1667] [Ideas]
In my distribution, pmount package wasn't updated since 2018 and I didn't want to install it. so I changed **pmount** command with **udisksctl -b mount** and **pumount** with **udiskctl -b umount**. For me it works well.
I believe it would be better with less dependencies.

**Comment by jarun** [comment]
pmount has its benefits. I would suggest you compile and use the latest if you want to use it.
Plugins are indicative. If an alternative works for you, have it.
